ui/app/AppLayouts/Chat/views/private/MembersSelectorBase.qml

```diff
@@ -50,9 +52,13 @@ InlineSelectorPanel {
             return true
         }
 
+        function isPastedProfileLinkToContact(pubkey) {
+            return root.pastedChatKey === pubkey
+        }
+
         filters: [
             ExpressionFilter {
-                enabled: root.edit.text !== ""
+                enabled: root.edit.text !== "" && root.pastedChatKey == ""
```

Comment:
```suggestion
                enabled: root.edit.text !== "" && root.pastedChatKey === ""
```

Reply:
No, its no needed bc of both string types :) no warnings

Reply:
It was not really about warnings. It's just good practice in JS to always use `===` because it's safer

Reply:
yes I understand how it works, I can push new PR to fix it, especially, I think `!root.pastedChatKey` is enough :)